wagtaildraftail/widgets.py

```diff
@@ -56,7 +55,12 @@ def render(self, name, json_value, attrs=None):
 
         encoded_value = json.dumps(value)
 
-        return super(DraftailTextArea, self).render(name, encoded_value, attrs)
+        parent = super(DraftailTextArea, self)
+
+        if func_supports_parameter(parent.render, 'renderer'):  # >= Django 1.11
+            return parent.render(name, encoded_value, attrs, renderer)
```

Comment:
well, it's just more clunky to use naming convention for just 4 arguments. and doesn't seem to be changing any time soon too